ocaml/utils/consistbl.ml

```diff
@@ -60,6 +60,11 @@ end) = struct
 
   let source tbl name = snd (Module_name.Tbl.find tbl name)
 
+  let find t name =
```

Comment:
I think it's worth updating `extract`  and `extract_map` to use this function.

Reply:
Fixed